src/panels/config/users/ha-user-picker.js

```diff
@@ -0,0 +1,109 @@
+import '@polymer/paper-fab/paper-fab.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-item/paper-item-body.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../layouts/hass-subpage.js';
+
+import LocalizeMixin from '../../../mixins/localize-mixin.js';
+import NavigateMixin from '../../../mixins/navigate-mixin.js';
+
+
+let registeredDialog = false;
+
+/*
+ * @appliesMixin LocalizeMixin
+ * @appliesMixin NavigateMixin
+ */
+class HaUserPicker extends NavigateMixin(LocalizeMixin(PolymerElement)) {
+  static get template() {
+    return html`
+  <style include="iron-flex ha-style">
+    paper-fab {
+      position: fixed;
+      bottom: 16px;
+      right: 16px;
+      z-index: 1;
+    }
+    paper-card {
+      display: block;
+      max-width: 600px;
+      margin: 16px auto;
+    }
+    a {
+      color: var(--primary-text-color);
+    }
+  </style>
+
+  <hass-subpage header="[[localize('ui.panel.config.users.picker.title')]]">
+    <paper-card>
+      <template is="dom-repeat" items="[[users]]" as="user">
+        <a href='[[_computeUrl(user)]]'>
+          <paper-item>
+            <paper-item-body two-line>
+              <div>[[_withDefault(user.name, 'Unnamed User')]]</div>
+              <div secondary="">[[user.id]]</div>
+            </paper-item-body>
+            <iron-icon icon="hass:chevron-right"></iron-icon>
+          </paper-item>
+        </a>
+      </template>
+    </paper-card>
+
+    <paper-fab
+      is-wide$="[[isWide]]"
+      icon="hass:plus"
+      title="[[localize('ui.panel.config.users.picker.add_user')]]"
+      on-click="_addUser"
+    ></paper-fab>
+  </hass-subpage>
+`;
+  }
+
+  static get properties() {
+    return {
+      hass: Object,
+      users: Array,
+
+    };
+  }
+
+  ready() {
+    super.ready();
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    if (!registeredDialog) {
+      registeredDialog = true;
+      this.fire('register-dialog', {
+        dialogShowEvent: 'show-add-user',
+        dialogTag: 'ha-dialog-add-user',
+        dialogImport: () => import('./ha-dialog-add-user.js'),
+      });
+    }
+  }
+
+  _withDefault(value, defaultValue) {
+    return value || defaultValue;
+  }
+
+  _computeUrl(user) {
+    return `/config/users/${user.id}`;
+  }
+
+  _addUser() {
+    this.fire('show-add-user', {
```

Comment:
EventsMixin not required here?

Reply:
Good catch. It's part of the navigate mixin so that's why it worked.